docker/helm-chart/strategies/Strategy001.py

```diff
@@ -0,0 +1,80 @@
+# --- Do not remove these libs ---
```

Comment:
We'll clearly not want the strategies in this repo.

The sample-strategies are intentionally separated into a seperate repository - as they should serve as samples (and that should become clear through that).